pandas/core/common.py

```diff
@@ -454,3 +454,21 @@ def _pipe(obj, func, *args, **kwargs):
         return func(*args, **kwargs)
     else:
         return func(obj, *args, **kwargs)
+
+
+def get_rename_function(mapper):
+    """
+    Returns a function that will map names/labels, dependent if mapper
+    is a dict, Series or just a function.
+    """
+    if isinstance(mapper, (dict, ABCSeries)):
+
+        def f(x):
+            if x in mapper:
+                return mapper[x]
```

Comment:
Is this if / else equivalent to `dict.get(x, x)`?

Reply:
Maybe so, but I was preserving existing code.  That particular snippet is from a commit on October 21, 2011 (https://github.com/pandas-dev/pandas/commit/cb819780b598cbfe885a8040a2db32fa103bdcfb#diff-150fd3c5a732ae915ec47bc54a933c41 if you care to look)

So the question is whether I make that change now, or leave working code alone.